ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -352,6 +353,16 @@ public JSONObject toJson() throws JSONException {
     metadata.put(DESCRIPTION_KEY, description);
     metadata.put(IS_PRIVATE_KEY, !cacheable);
     metadata.put(PARENT_ACCOUNT_ID_KEY, parentAccountId);
+    // code to write in V2 (will be enabled once read support is fully deployed)
```

Comment:
keep this under a switch case? Not sure why this needs to be commented out

Reply:
done